### DISCUSSION TITLE
OnResourceLoadComplete not working

### DISCUSSION THREAD

**Discussion by PHOLAN** [cefsharp/CefSharp#3425] [General]
Have been using CefSharp for a few years. I extensively use IREquestHandler.OnResourceLoadComplete to intercept messages from the website to the host.
It seems that in the latest release, this is no longer working as I can't get it to trigger.

Thanks

**Comment by amaitland** [comment]
> CEF now supports the Chromium Network Service: If you are upgrading from a version prior to 75.1.x please make sure you read #2743.

As per https://github.com/cefsharp/CefSharp/releases/tag/v88.2.90

You will need to migrate from IRequestHandler.OnResourceLoadComplete to IResourceRequestHandler.OnResourceLoadComplete

See https://github.com/cefsharp/CefSharp/wiki/General-Usage#request-interception for a basic example of using the new ResourceRequestHandler implementation. 

In future please use https://github.com/cefsharp/CefSharp/discussions to ask questions.

**Reply by PHOLAN** [reply]
That didn't work either but i found a great post from you in stack overflow that solved my issue using CustomResourceRequestHandler
